orttraining/orttraining/python/training/onnxblock/engine/training_module.py

```diff
@@ -0,0 +1,64 @@
+from onnxruntime.capi import _pybind_state as C
+
+
+class TrainingModule:
+    """
+    Class for running Training.
+    This class is a wrapper of Module Class.
+    """
+
+    def __init__(
+        self, train_model_uri, ckpt_uri, eval_model_uri=None, env=None, session_options=None, providers=None, **kwargs
```

Comment:
A single env instance is maintained within pybind, so the env arg should be dropped.